gold/tmp_coeff.m

```diff
@@ -0,0 +1,42 @@
+generate_data( diffusion1, 'diffusion1', 'lev', 2, 'deg', 2, 'grid_type', 'FG' );
```

Comment:
what are these `tmp_` files for?

Reply:
I wanted to add a little bit of code to a file that already had a lot of stuff in it - so I put the new stuff in its own file while developing so that I didn't have to run the whole thing